presto-main/src/main/java/io/prestosql/metadata/Split.java

```diff
@@ -62,7 +64,11 @@ public Lifespan getLifespan()
 
     public Object getInfo()
     {
-        return connectorSplit.getInfo();
+        return ImmutableMap.builder()
+                .put("catalogName", catalogName)
```

Comment:
This should be a field in SplitCompletedEvent, instead.

Reply:
Modified. I've added it to `SplitOperatorInfo` as well, so that it's accessible during `SplitCompletedEvent` creation. Getting catalog name from a "stats" object is a bit indirect, but I felt that this is the simpler, even though there's redundancy. otherwise, we could pass the optional catalog from `SqlTaskExecution#enqueueDriverSplitRunner`. wdyt?